com.unity.multiplayer.mlapi/Prototyping/NetworkAnimator.cs

```diff
@@ -541,7 +549,8 @@ private void ApplyAnimatorSnapshot(AnimatorSnapshot animatorSnapshot)
                 var currentAnimatorState = m_Animator.GetCurrentAnimatorStateInfo(layerIndex);
 
                 bool stateChanged = currentAnimatorState.fullPathHash != layerState.StateHash;
-                bool forceAnimationCatchup = !stateChanged && Mathf.Abs(currentAnimatorState.normalizedTime - currentAnimatorState.normalizedTime) >= 0.15f;
+                bool forceAnimationCatchup = !stateChanged &&
```

Comment:
So if I understand this correctly this forced catch up was never used before because `currentAnimatorState.normalizedTime - currentAnimatorState.normalizedTime` is well always `0`?

Reply:
Affirmative.